src/group_cmd.cpp

```diff
@@ -355,7 +355,7 @@ std::tuple<CommandCost, GroupID> CmdCreateGroup(DoCommandFlag flags, VehicleType
 		if (pg == nullptr) {
 			g->livery.colour1 = c->livery[LS_DEFAULT].colour1;
 			g->livery.colour2 = c->livery[LS_DEFAULT].colour2;
-			if (c->settings.renew_keep_length) SetBit(g->flags, GroupFlags::GF_REPLACE_WAGON_REMOVAL);
+			if (c->settings.renew_keep_length) g->flags |= GroupFlags::ReplaceWagonRemoval;
```

Comment:
I wonder if it's worth abstracting `SetGroupFlag()` to a more generic `SetFlag()` to use here and in afterload. Maybe you have a patch for that rabbithole already. 😉 

(That should not prevent this PR, can be done separately or not at all.)